doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,169 @@
+
+Goals:
+
+- Create a guided prompt that exposes the user to a brief overview of each PowerToy, highlighting the PowerToys’ basic usage.
+- Present a short-list of useful customization options that the user can select to define their experience.
+
+Non-goals:
+
+- Present an exhaustive list of features and options that are available in PowerToys. This OOBE prompt is only meant to provide initial exposure for which interested users should continue investigating via provided links to documentation and settings.
+
+## 2. Definition of Success
+
+### 2.1. Customers
+
+The PowerToys OOBE is for new power users and developers who are looking to tune and streamline their Windows experience for greater productivity and enhanced user experience. As the PowerToys customer base tends to be particularly biased against OOBE prompts in general, we need to present the PowerToys OOBE dialog in such a way that it provides immediate value to end-users to improve the likelihood of users discovering all the PowerToys by completing the prompts.
+
+### 2.2. Expected Impact: Customer, and Technology Outcomes
+
+- **High Reliability:** Less than 0.1% crash rate.
+- **Increased Activation:** 30% or more increase in usage in all PowerToys features.
+- **High User Retention:** 25% or more active PowerToys users after 28 days.
+- **High OOBE Completion Rate:** 50%  or more of New PowerToys users reach the last page of the OOBE prompt.
+
+## 3. Requirements
+
+The base of this OOBE dialog was attributed by [Niel’s mock-up](https://github.com/microsoft/PowerToys/issues/1285).
+
+### 3.1. Functional Requirements
+
+#### 3.1.1. Functional Requirements Overview
+
+|No. | Requirement | Pri |
+| - | - | - |
+|1 | The OOBE Dialog window should launch immediately after installation. | P0 |
+|2 | The OOBE Dialog should consist of a collection of pages that describe the available PowerToys. | P0 |
+|3 | On launch, present the Welcome/Landing page in the OOBE Dialog window based on the template described in [Figure 5.1.1.](#511-oobe-dialog-layout) | P1 |
+|4 | Traverse the various OOBE Dialog pages via an list view options on the left-hand pane. | P0 |
+|5 | Alternatively traverse the various OOBE Dialog pages via an on-screen next/previous buttons on each page. | P1 |
+|6 | At any time, the user should be able to exit the OOBE dialog by closing the window. | P0 |
+|7 | Alternatively, the user should be able to exit the OOBE dialog by pressing the ESC key. | P1 |
+|8 | When the OOBE dialog is exited, save all the options that were set by the user. | P0 |
+|9 | In the PowerToys settings “General” menu, add a button to re-launch the OOBE dialog under the “Run at startup” option. | P0 |
+|10 | The OOBE dialog window should be able to be enlarged to accommodate accessibility needs. | P0 |
+|11 | The OOBE dialog should have a preset minimum window size to ensure all content can be viable with the need to scroll the page. | P1 |
+
+#### 3.1.2. Page Layout
+
+| No. | Requirement | Pri |
```

Comment:
Agreed. I think the how to Run/Launch section needs to emphasize that users should "try it out" to help improve engagement based on the user research study feedback.

Per my response to Niels comment earlier, I'm not sure if the key visualization will drive clarity/utilization more than just having a textual representation. Definitely more aesthetically pleasing, but not critical, so I'll give it a lower priority unless you feel otherwise.